fhir-persistence-schema/docs/tests/QueryText.md

```diff
@@ -0,0 +1,28 @@
+# Scratch Area for Query Development
```

Comment:
hum... should there be a copyright here too?

Reply:
removed